service/src/main/java/com/epam/rft/atsy/service/impl/CandidateServiceImpl.java

```diff
@@ -183,10 +182,18 @@ private void validateCandidateFilterRequest(CandidateFilterRequest candidateFilt
 
       try {
         Sort.Direction.fromString(candidateFilterRequest.getSortOrder());
-      } catch (IllegalArgumentException e) {
+      } catch (Exception e) {
         throw new IllegalArgumentException(
             "Invalid sort order: " + candidateFilterRequest.getSortOrder(), e);
       }
+
+      try {
```

Comment:
This block handles the validation of the page request, e.g.: if the sortName  argument is malformed an exception is thrown.